docs/teams/phoenix_integration.md

```diff
@@ -0,0 +1,298 @@
+
+```elixir
+phoenix_app_node = :"my_app@127.0.0.1"
+phoenix_app_cookie = :secret
+
+Node.set_cookie(phoenix_app_cookie)
+Node.connect(phoenix_app_node)
+```
+
+Now you can call functions from your Phoenix app using [`erpc`](https://www.erlang.org/doc/apps/kernel/erpc.html):
+
+```elixir
+# Calling a function from your Phoenix app
+:erpc.call(phoenix_app_node, MyApp.Accounts, :get_user, [123])
+```
+
+> #### Understanding the multiple Erlang VM nodes involved {: .info}
+>
+By default, Livebook starts a new Erlang VM node for each notebook. This is
+> the [standalone runtime](runtime.md#standalone-runtime).
+>
+> Since your Phoenix app is running on another node, you need to use remote procedure
+> calls to execute functions from your Phoenix app:
+>
+> ```mermaid
+> graph LR
+>     subgraph livebook_node["Erlang VM node"]
+>         A[Livebook]
+>     end
+>
+>    subgraph standalone_node["Erlang VM node Livebook started for your notebook"]
+>         B[code inside the notebook]
+>     end
+>
+>    subgraph app_node["Erlang VM node running your Phoenix app"]
+>         C[Phoenix app]
+>     end
+>
+>    A -.-|starts and clusters with| standalone_node
+>     B -.->|remote procedure call| C
+> ```
+
+## Connect to your Phoenix app in production
+
+When [developing and deploying a Livebook app](deploy_app.md) that integrates with a Phoenix app, you need a way
+to handle the connection between them during both development and production.
+
+To support both environments, use Livebook secrets to make the connection between your notebook
+and your Phoenix app configurable.
+
+### Set up environment secrets
+
+Create these secrets in your Livebook Teams workspace:
+
+**For development:**
+- `PHOENIX_APP_ENV`: Set to `dev`
+- `PHOENIX_APP_COOKIE`: Set to your local cookie value (e.g., `secret`)
+
+![](images/clustering_dev.png)
+
+**For production:** use additional secrets in your Teams workspace deployment group to override these values:
+
+- `PHOENIX_APP_ENV`: Set to `production`
+- `PHOENIX_APP_COOKIE`: Set to your production cookie value
+
+![](images/clustering_config_per_env.png)
+
+### Create a connection module
+
+Add this module to your notebook to handle environment-specific connections:
+
+```elixir
+defmodule NodeConnection do
+  def connect() do
+    Node.set_cookie(cookie())
+
+    case Node.connect(target_node()) do
+      true -> :ok
+      _ -> {:error, "Failed to connect to #{inspect(target_node())}"}
+    end
+  end
+
+  def cookie() do
+    String.to_atom(System.fetch_env!("LB_PHOENIX_APP_COOKIE"))
+  end
+
+  def target_node() do
+    case System.fetch_env!("LB_PHOENIX_APP_ENV") do
+      "dev" ->
+        :"my_app@127.0.0.1"
+      env when env in ["staging", "production"] ->
+        discover_node()
+    end
+  end
+
+  defp discover_node() do
+    # Implementation depends on your deployment platform
+    # See platform-specific examples below
+  end
+end
+
+# Connect to your Phoenix app
+NodeConnection.connect()
```

Comment:
Should we tell them they can place this on a file alongside their notebooks and then `Code.require_file` it?